source/_integrations/clarifai.markdown

```diff
@@ -0,0 +1,210 @@
+---
+title: Clarifai
+description: Instructions on how to integrate Clarifai with Home Assistant.
+ha_category:
+  - Image Processing
+ha_release: 0.112
+ha_domain: clarifai
+ha_codeowners:
+  - '@djbowers'
+---
+
+The `clarifai` integration allows you to use the
+[Clarifai](https://www.clarifai.com/) API through Home Assistant.
+To get started, create a free account on the Clarifai [Portal](https://portal.clarifai.com/signup).
+
+For step-by-step instructions on creating a model using the Clarifai Portal,
+visit the Clarifai [docs](https://docs.clarifai.com/portal-guide/portal_overview).
+
+## Setup
+
+To authenticate with the Clarifai API, users must first create a [Personal Access Token](https://docs.clarifai.com/getting-started/authentication/personal-access-tokens)
+using the Clarifai Portal, then save that token to their Home Assistant configuration file.
+
+## Configuration
+
+To enable the Clarifai integration,
+add the following snippet to your `configuration.yaml` file:
+
+```yaml
+# To set up the Clarifai integration
+clarifai:
+  access_token: YOUR_PERSONAL_ACCESS_TOKEN
+```
+
+{% configuration %}
+access_token:
+  description: The Personal Access Token used to access your Clarifai applications.
+  example: 84a83d6a6ba241f5905a8971ea55cf48
```

Comment:
I sure hope this is not an actual access token? :)

Reply:
It's not, I made sure I got a `11009 API key not found` for it, though I suppose that doesn't necessarily mean it couldn't become valid one day, so maybe I should change that still?

Reply:
I think you can get rid of the example line altogether. You are already showing the example above anyway.
```suggestion
```